lib/pf/radius.pm

```diff
@@ -694,6 +694,64 @@ sub logger {
     return get_logger( ref($proto) || $proto );
 }
 
+=item switch_access
+
+return radius attributes or reject for switch login
+
+=cut
+
+sub switch_access {
+    my ($self, $radius_request) = @_;
+    my $logger = $self->logger;
+    my $start = Time::HiRes::gettimeofday();
+    my($switch_mac, $switch_ip,$source_ip,$stripped_user_name,$realm) = $self->_parseRequest($radius_request);
+
+    $logger->debug("instantiating switch");
+    my $switch = pf::SwitchFactory->instantiate({ switch_mac => $switch_mac, switch_ip => $switch_ip, controllerIp => $source_ip});
+
+    # is switch object correct?
+    if (!$switch) {
+        $logger->warn(
+            "Can't instantiate switch ($switch_ip). This request will be failed. "
```

Comment:
I'm tired of seeing that message :P

Change it to 'This request will fail' or something that has a valid grammar